recipe/build-pkg.sh

```diff
@@ -19,25 +19,26 @@ cmake ${CMAKE_ARGS} \
 cmake --build _build_python_generic --target swigfaiss -j $CPU_COUNT
 
 # Build version with avx2 support, see build-lib.sh
-cmake ${CMAKE_ARGS} \
-    -Dfaiss_ROOT=_libfaiss_avx2_stage/ \
-    -DFAISS_OPT_LEVEL=avx2 \
-    -DFAISS_ENABLE_GPU=${FAISS_ENABLE_GPU} \
-    -DCMAKE_BUILD_TYPE=Release \
-    -DPython_EXECUTABLE="${PYTHON}" \
-    -B _build_python_avx2 \
-    faiss/python
-cmake --build _build_python_avx2 --target swigfaiss_avx2 -j $CPU_COUNT
+if [[ "${target_platform}" == *-64 ]]; then
```

Comment:
Is this supposed to match `win-64` and `linux-64` but not `linux-aarch64` or `linux-ppc64le`? What about `osx-64`? I saw a comment below, `the linux & windows CI agents support AVX2 (OSX doesn't yet)`. I'd prefer to check explicitly for specific matching strings.

Reply:
This isn't used on Windows. There are `.bat` build files for that.

AFAICT this was already building on macOS. So this doesn't change that.

All this does is exclude non-`x86_64` targets.